src/tplink_omada_client/devices.py

```diff
@@ -191,7 +192,7 @@ def poe_power(self) -> float:
         """Power (W) supplied over PoE."""
         if "poePower" in self._data:
             return self._data["poePower"]
-        return 0.0
+        return None
 
```

Comment:
The `if` statement could be removed by simply doing:
```python
    return self._data.get("poePower")
```
The return type hint should also be changed to `Optional[float]` to indicate a `None` may be returned.

Reply:
I only learned about it recently myself. :)